java/tests/com/google/privacy/differentialprivacy/BoundedSumTest.java

```diff
@@ -678,4 +678,213 @@ private static void testForBias(
+            .maxPartitionsContributed(1)
+            .lower(1)
+            .upper(5)
+            .build();
+    sum.addEntry(10);
+    Exception exception = assertThrows(IllegalStateException.class, () -> {
+      sum.computeConfidenceInterval(0.1554684);
+    });
+    assertThat(exception)
+            .hasMessageThat()
+            .startsWith("Noised sum must be computed before calling this function.");
+  }
+
+  @Test
+  public void computeConfidenceInterval_whenComputeResultWasNotCalled_throwsException_forDouble() {
+    // Mock the noise mechanism. Since noise is not Laplace, nor Gaussian, delta will be passed as a value instead of null, in order to pass the checks.
+    when(noise.computeConfidenceInterval(anyDouble(), anyInt(), anyDouble(), anyDouble(), anyDouble(), anyDouble()))
+            .thenAnswer(invocation -> new LaplaceNoise().computeConfidenceInterval(
+                    (Double) invocation.getArguments()[0],  (Integer) invocation.getArguments()[1], (Double) invocation.getArguments()[2],
+                    (Double) invocation.getArguments()[3], null,  (Double) invocation.getArguments()[5]));
+    BoundedSum.builder()
+            .epsilon(EPSILON)
+            .delta(DELTA)
+            .noise(noise)
+            .maxPartitionsContributed(1)
+            .lower(1)
+            .upper(5)
+            .build();
+    sum.addEntry(10.58687487);
+    Exception exception = assertThrows(IllegalStateException.class, () -> {
+      sum.computeConfidenceInterval(0.1554684);
+    });
+    assertThat(exception)
+            .hasMessageThat()
+            .startsWith("Noised sum must be computed before calling this function.");
+  }
+
+  @Test
+  public void computeResult_callsNoiseCorrectly_ForConfidenceIntervals() {
+    double alpha = 0.1524;
+    when(noise.computeConfidenceInterval(anyDouble(), anyInt(), anyDouble(), anyDouble(), anyDouble(), anyDouble()))
+            .thenAnswer(invocation -> new GaussianNoise().computeConfidenceInterval(
+                    (Double) invocation.getArguments()[0],  (Integer) invocation.getArguments()[1], (Double) invocation.getArguments()[2],
+                    (Double) invocation.getArguments()[3],  (Double) invocation.getArguments()[4],  (Double) invocation.getArguments()[5]));
+    sum =
+            BoundedSum.builder()
+                    .epsilon(EPSILON)
+                    .delta(DELTA)
+                    .noise(noise)
+                    .maxPartitionsContributed(1)
+                    .maxContributionsPerPartition(1)
+                    .lower(0)
+                    .upper(100)
+                    .build();
+    sum.computeResult();
+    ConfidenceInterval confInt = sum.computeConfidenceInterval(alpha);
+
+    verify(noise)
+            .computeConfidenceInterval(
+                    eq(0.0),
+                    eq(1),
+                    eq(1.0),
+                    eq(EPSILON),
+                    eq(DELTA),
+                    eq(alpha));
+  }
+  }
```

Comment:
looks like something is wrong with the formatting, please re-format the whole file